src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Config/QueuesExtensionConfigProvider.cs

```diff
@@ -6,24 +6,24 @@
 using System.Reflection;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Description;
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.Azure.WebJobs.Host.Config;
 using Microsoft.Azure.WebJobs.Host.Protocols;
 using Microsoft.Azure.WebJobs.Host.Queues.Triggers;
 using Microsoft.WindowsAzure.Storage.Queue;
 using Newtonsoft.Json.Linq;
 
-namespace Microsoft.Azure.WebJobs.Host.Queues.Bindings
+namespace Microsoft.Azure.WebJobs.Host.Queues.Config
 {
-    // Write up bindinging rules for [Queue] attribute. 
-    // This is fundemantentally an IAsyncCollector<IStorageQueueMessage>
-    internal class QueueExtension : IExtensionConfigProvider
+    [Extension("AzureQueues")]
+    internal class QueuesExtensionConfigProvider : IExtensionConfigProvider
```

Comment:
TODO : This is the other one that can't be made public due to non-public ctor params